vendor/karibbu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6.9.4-alpine
+FROM rezzza/docker-node:6.9.4
```

Comment:
BTW it's based on alpine 3.5 from now, so goodbye to apk weird things (3.4/edge)